sk/static/nyaablue.user.js

```diff
@@ -39,22 +40,49 @@ async function setViewBlue () {
 
   const { items } = await torrentsResponse.json()
   if (items[0]?.infoHash) {
-    element?.classList.add('panel-info')
+    element?.classList.add(items[0]?.isBest ? 'panel-info' : 'panel-info-alt')
+  }
+  if (items[0]?.id) {
+    const collectionResponse = await seadexEntryEndpoint('', {
+      filter: 'trs~"' + items[0]?.id + '"'
```

Comment:
why.... not infoHash search? %LIKE% is fucking slow

Reply:
This isn't torrents. This is entries.